go.mod

```diff
@@ -4,12 +4,10 @@ go 1.13
 
 require (
 	github.com/cockroachdb/cockroach-go/v2 v2.0.6
-	github.com/denisenkom/go-mssqldb v0.0.0-20200620013148-b91950f658ec // indirect
-	github.com/go-pg/pg/v9 v9.1.6
-	github.com/go-sql-driver/mysql v1.5.0 // indirect
-	github.com/jinzhu/gorm v1.9.12
-	github.com/jinzhu/now v1.1.1 // indirect
+	github.com/go-pg/pg/v10 v10.9.0
 	github.com/julienschmidt/httprouter v1.1.0
 	github.com/lib/pq v1.4.0
```

Comment:
perhaps we should upgrade lib/pq to 1.10.0

and any other drivers versions that are pinned in these tests?

Reply:
Good point, done. I didn't find any other pinned drivers.